app/src/main/java/com/example/solar_alarm/Alarm.java

```diff
@@ -4,21 +4,24 @@
 
 import androidx.annotation.RequiresApi;
 
+import java.io.Serializable;
 import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
 import java.util.ArrayList;
 
-public class Alarm
+public class Alarm implements Serializable
 {
-    private LocalTime AlarmTime;
+    private static ArrayList<Alarm> contacts = new ArrayList<Alarm>(1);
+    private String AlarmTime;
```

Comment:
Keep this as a `LocalTime` and convert it to a string when you need to display it.